Package.swift

```diff
@@ -144,6 +144,10 @@ let package = Package(
         ),
     ],
     targets: packageCollectionsSigningTargets + [
+        .target(
+            name: "SQLite3",
+            exclude: ["CMakeLists.txt"]
+        ),
```

Comment:
I think we should also make a note of this system dependency

Reply:
this should also be moved to just above or after "Basics" target description to honor the comment in L175